src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/handler/PaymentMessageHandler.java

```diff
@@ -78,7 +70,40 @@ public void updatePaymentCaseReference(UpdatePaymentMessage paymentMessage) {
             paymentMessage.envelopeId,
             response.getStatusCode()
         );
-
     }
 
+    private void createPayment(CreatePaymentMessage paymentMessage, String messageId) {
+        CreatePaymentRequest request = paymentRequestMapper.mapPaymentMessage(paymentMessage);
+
+        log.info(
+            "Sending Payment request with Document Control Numbers: {}, Envelope id: {}, poBox: {}",
+            String.join(", ", request.documentControlNumbers),
+            paymentMessage.envelopeId,
+            paymentMessage.poBox
+        );
+
+        try {
+            CreatePaymentResponse paymentResult = payHubClient.createPayment(
+                authTokenGenerator.generate(),
+                request
+            ).getBody();
+
+            log.info(
+                "Payment response received from PayHub: {}. Envelope id: {}. Ccd case reference: {}",
+                paymentResult == null ? null : String.join(", ", paymentResult.paymentDcns),
+                paymentMessage.envelopeId,
+                paymentMessage.ccdReference
+            );
+        } catch (PayHubClientException ex) {
+            if (ex.getStatus() == HttpStatus.CONFLICT) {
```

Comment:
This bit has been moved from the caller class, as we need to know at this level whether to proceed with CCD update or fail completely.